test/dlc_tests/container_tests/bin/testTF2HVD.py

```diff
@@ -48,7 +48,11 @@
 loss = tf.losses.SparseCategoricalCrossentropy()
 
 # Horovod: adjust learning rate based on number of GPUs.
-opt = tf.optimizers.Adam(0.001 * hvd.size())
+if tf.__version__ in SpecifierSet("<2.11.0"):
```

Comment:
Please confirm that `x in SpecifierSet(y)` works in the same way as `Version(x) in SpecifierSet(y)`.

Reply:
Will update this to use Version(x) to keep consistent with other instances of this check.